clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookDownloadServiceTest.spec.ts

```diff
@@ -63,6 +63,7 @@ const getZoweExplorerMock = (forError: boolean = false, mentionedError?: any) =>
             apiResponse: {
                 items: [
                     { name: "uss_copybook" },
+                    { name: "uss_withExt.cpy"},
```

Comment:
How about a file with multiple suffixes? Would that test something currently not covered?

Reply:
Good point. A USS directory with SUBSCHEMA-NAMES.cpy and SUBSCHEMA-NAMES.bkp may result in wrong file downloading. But I don't think there is an option to resolve this until we introduce a setting to identify the supported extension as the COBOL src code only mentions the filename.
But if, have a directory with copybooks like C1.cpy and C2.copy. These would be resolved.

Reply:
By multiple suffixes, you mean, ABC.cpy.bkp.  Ahh.. this would fail. If this is a valid scenario. I can work on it.